### PR TITLE
feat : added cards component

### PR DESCRIPTION
**Pull request by Priya-Patel31**

### CODE REVIEW

**Review by munjalSaksham71 — APPROVED** [review]
Looks Good! You can merge it off.

File: component/card/card.css [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  .vertical-card-content h4 {
+    margin: 0.5rem 0;
+    font-weight: 400;
+  }
+  .badge-trending {
+    position: absolute;
+    /* opacity: 0.5; */
+    top: 0.8rem;
+    left: 0px;
+    width: 130px;
```
Comment (rkapoor10) [inline component/card/card.css:77]: give in rems maybe

File: component/card/card.css [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+      font-weight: 700;
+      padding:0.5rem;
+      border-radius:1rem;
+      background-color:var(--light-pink);
+      color:var(--light-color);
+      border:none;
+      cursor: pointer;
+  }
+  
+
```
Comment (rkapoor10) [inline component/card/card.css:138]: just a suggestion..you can have all button-related CSS in buttons.css.

File: component/card/card.css [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  
+  .card-hor-text-ctn {
+    flex: 1;
+    padding: 1rem;
+    box-sizing: border-box;
+  }
+  .card-hor-image-ctn {
+    max-width: 170px;
+    height: auto;
+  }
```
Comment (Ankur9669) [inline component/card/card.css]: You have created (card-hor-image-ctn) class twice.
See on line 11

**Review by rkapoor10 — APPROVED** [review]

File: component/card/card.css [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+  .card-img-hor {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+  }
+  .discount-price {
+    font-size: 17px;
```
Comment (Ankur9669) [inline component/card/card.css:32]: You can use rems here

File: component/card/card.css [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+    font-weight: 600;
+  }
+  .card-hor-btn {
+    margin-top: 6px;
+  }
+  .original-price {
+    text-decoration: line-through;
+    font-size: 14px;
```
Comment (Ankur9669) [inline component/card/card.css:40]: You can use rems here

File: component/card/card.css [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+      position: relative;
+  }
+  .vertical-image-ctn {
+    flex: 1;
+    position: relative;
+  }
+  .vertical-image {
+    width: 100%;
+    height: 100%;
+    border-radius: 0.5rem;
```
Comment (Ankur9669) [inline component/card/card.css:63]: You can add object-fit property

File: component/card/card.css [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+    left: 0px;
+    width: 130px;
+    clip-path: polygon(0% 0%, 71% 0%, 53% 50%, 69% 97%, 0 97%);
+  }
+  .vertical-card-outer-ctn{
+    display: flex;
+    gap:4rem;
+    flex-wrap:wrap;
+    
+  }
```
Comment (Ankur9669) [inline component/card/card.css:84]: You can remove white space here

File: component/card/card.css [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+    font-size: 1.5rem;
+    position: absolute;
+    top:-15px;
+    right:-15px;
+  }
+  .dismiss-card-ctn img,.dismiss-card-ctn{
+    border-radius:0;
+  }
+  .text-overlay-ctn {
+    opacity: 0.5;
```
Comment (Ankur9669) [inline component/card/card.css:98]: This looks fine but you can use rgba here.

File: component/card/card.css [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+      font-weight: 1000;
+      top:40%;
+      z-index:2;
+  }
+  .text-overlay{
+       position: absolute;
+       top:50%;
+       left:50%;
+       transform:translate(-50%,-50%);
+       font-weight: 800;
```
Comment (Ankur9669) [inline component/card/card.css:113]: Font-weight is not working, so you can safely remove it

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
Most of the things are good, so great work👏👏.

File: component/card/card.css (hunk 1 above)
Comment (Priya-Patel31) [reply component/card/card.css:77]: okk i will change !!

File: component/card/card.css (hunk 2 above)
Comment (Priya-Patel31) [reply component/card/card.css:138]: yeahh making changes over it !!

File: component/card/card.css (hunk 3 above)
Comment (Priya-Patel31) [reply component/card/card.css]: made changes in it !!

**Review by Ankur9669 — APPROVED** [review]
Great work!!!👏👏👏. Go ahead and merge it

**Pull request merged**